Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM python:3.7.9
 
 RUN apt-get update && apt-get -y install libpq-dev
+RUN apt-get install -y nodejs
+RUN apt-get install -y npm
```

Comment:
@moz-astults Do you have an opinion on which version? I have the following locally. 

```
node: v10.20.1
npm: 6.14.4
```

Reply:
```suggestion
RUN apt-get install -y nodejs=v10.20.1
RUN apt-get install -y npm=6.14.4
```